app/pages/amigos/amigos.html

```diff
@@ -1,5 +1,9 @@
 <dose-amigos-toolbar [title]="title"></dose-amigos-toolbar>
 
 <ion-content padding class="amigos">
-  <div>Hola Amigos!</div>
+
+    <user-status-card *ngFor="let user of doseAmigosUsers"
+                      [user]="user"
+                      [onClickPage]="userStatusClickPage"></user-status-card>
```

Comment:
Uses the same user-status-card we show at the top of the News Feed page for each of the current user's Amigos.

This allows a quick at-a-glance view of if each of your Amigos is on track:

<img width="377" alt="amigos" src="https://cloud.githubusercontent.com/assets/6886583/16435707/f14ac0b4-3d65-11e6-89dc-657184b9f386.png">